.autopep8.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# TODO: Explain usage here?  Why do we need both pylint and pep8?
```

Comment:
PEP8 is a Python formatting guideline. Autopep8 is a tool to automatically format python code according to the PEP8 standards. Pylint is a tool to lint python code.

Reply:
Okay, pep8 for format, separately from linting.